griptape/drivers/image_query/anthropic_image_query_driver.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Optional
+from attr import define, field, Factory
+from griptape.artifacts import ImageArtifact, TextArtifact
+from griptape.drivers import BaseImageQueryDriver
+from griptape.utils import import_optional_dependency
+
+if TYPE_CHECKING:
+    from anthropic import Anthropic
+
+
+@define
+class AnthropicImageQueryDriver(BaseImageQueryDriver):
+    """
+    Attributes:
+        api_key: Anthropic API key.
+        model: Anthropic model name.
+        client: Custom `Anthropic` client.
+        max_output_tokens: Max output tokens to return.
+    """
+
+    api_key: str = field(kw_only=True, metadata={"serializable": True})
+    model: str = field(default="claude-3-sonnet-20240229", kw_only=True, metadata={"serializable": True})
+    client: Anthropic = field(
+        default=Factory(
+            lambda self: import_optional_dependency("anthropic").Anthropic(api_key=self.api_key), takes_self=True
+        ),
+        kw_only=True,
+    )
+    max_output_tokens: Optional[int] = field(default=4096, kw_only=True, metadata={"serializable": True})
+
+    def try_query(self, query: str, images: list[ImageArtifact]) -> TextArtifact:
+        content = []
+        for image in images:
+            content.append(self._construct_image_message(image))
+
+        content.append(self._construct_text_message(query))
+        messages = self._construct_messages(content)
+
+        params = {"model": self.model, "messages": messages}
+
+        if self.max_output_tokens is not None:
+            params["max_tokens"] = self.max_output_tokens
```

Comment:
Can we move this logic to a separate method like the Prompt Drivers have? Something like `OpenAiChatPromptDriver._base_params`